eservice/lib/libpdo_enclave/contract_compilation_report.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2020 Intel Corporation.
```

Comment:
I'm not sure what this refactoring would look like, or if it's really needed to make the compilation report/CDI usable in other applications, but I do agree that it might make sense to move the compilation report and enclave policy out of the eservice.

Reply:
I thought I was looking ahead in the future, but you are actually already using it in other applications -- lines 39-40 in `contracts/wawaka/cdi/CMakeLists.txt` -- because you need the report generation code in `gen-compilation-report`.

The refactoring makes a lot more sense now. In a nutshell, those two lines should not be there. `gen-compilation-report` should find everything it needs in `updo-common`. So, the compilation report would be compiled by itself as a static library, and then linked in `updo-common` (where it will find the crypto lib as well).